DigitalLearningSolutions.Data.Tests/DataServices/UserDataServiceTests/DelegateUserDataServiceTests.cs

```diff
@@ -272,5 +272,21 @@ public void DeactivateDelegateUser_deactivates_delegate_user()
             // then
             userDataService.GetDelegateUserById(1)!.Active.Should().BeFalse();
         }
+
+        [Test]
+        public void ReactivateDelegateUser_reactivates_delegate_user()
+        {
+            using var transaction = new TransactionScope();
+            const int delegateUserId = 29;
+
+            // given
+            userDataService.GetDelegateUserById(delegateUserId)!.Active.Should().BeFalse();
+
+            // when
+            userDataService.ReactivateDelegateUser(delegateUserId);
+
+            // then

```

Comment:
Minor, but could we set these comments to have a capitalised first letter to be consistent with most other tests? So they become Given, When, Test.

Reply:
sorry, I have updated the comments